### PR TITLE
[1.16] Remove pod infra exit file 

### PR DESCRIPTION
**Pull request by mrunalp**
@umohnani8 @haircommander ptal

### CODE REVIEW

File: server/server.go [hunk 1]
```diff
@@ -229,6 +229,37 @@ func (s *Server) restore() {
...
+			continue
+		} else {
+			sb := s.GetSandbox(exitID)
+			if sb != nil {
+				continue
+			}
+		}
+		logrus.Warnf("Removing exit file: %v", exitID)
+		if err := os.Remove(filepath.Join(exitDir, exitID)); err != nil && !os.IsNotExist(err) {
+			logrus.Warnf("Failed to remove container exit file during restore cleanup %s: %v", exitID, err)
```
Comment (TomSweeneyRedHat) [inline server/server.go]: I think it might more sense to have "Failed to remove container exit file %s during restore cleanup: %v"

**Comment by umohnani8** [comment]
Rebase to pick up CI fixes

**Comment by mrunalp** [comment]
rebased.

**Pull request merged**